Cannot find module log4js with TypeScript

Use the import method to import log4js, execute the tsc command, and the error "Cannot find module 'log4js'" will be reported, but the compiled js file can run normally.

error TS2792: Cannot find module 'log4js'
index.ts:
`import log4js from "log4js";
// log4js.configure({
//   appenders: { cheese: { type: "file", filename: "cheese.log" } },
//   categories: { default: { appenders: ["cheese"], level: "error" } }
// });

const logger = log4js.getLogger();
logger.level = "debug";
logger.debug("Some debug messages");`
![image](https://user-images.githubusercontent.com/52586868/213095409-7e3e8ebf-c476-411b-9add-204a4ef8f8cd.png)

--- a/README.md
+++ b/README.md
@@ -98,7 +98,7 @@ There's also [an example application](https://github.com/log4js-node/log4js-exam
 ## TypeScript
 
 ```ts
-import log4js from "log4js";
+import * as log4js from "log4js";
 log4js.configure({
   appenders: { cheese: { type: "file", filename: "cheese.log" } },
   categories: { default: { appenders: ["cheese"], level: "error" } },

